### PR TITLE
workout display 100% complete

### PR DESCRIPTION
**Pull request by EzyhB**

### CODE REVIEW

File: src/components/WorkoutDisplayCard/index.tsx [hunk 1]
```diff
@@ -32,11 +32,7 @@ export default function WorkoutDisplayCard() {
     }
     async function getAllExercise() {
       console.log('all exercises');
```
Comment (E-K8) [inline src/components/WorkoutDisplayCard/index.tsx]: Do we still need this console.log?

File: src/components/WorkoutDisplayCard/index.tsx [hunk 2]
```diff
@@ -48,11 +44,11 @@ export default function WorkoutDisplayCard() {
   };
   const handleClose = (el: workoutObj) => {
     console.log(el);
```
Comment (E-K8) [inline src/components/WorkoutDisplayCard/index.tsx]: same?

**Review by E-K8 — CHANGES_REQUESTED** [review]

File: src/components/WorkoutDisplayCard/index.tsx (hunk 1 above)
Comment (EzyhB) [reply src/components/WorkoutDisplayCard/index.tsx]: idk you were meant to tell me, "code reviewer"

File: src/components/WorkoutDisplayCard/index.tsx (hunk 2 above)
Comment (EzyhB) [reply src/components/WorkoutDisplayCard/index.tsx]: same

**Pull request merged**